docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+} 
+```
+
+## CRUD Operations
+
+Create, Read, Update and Delete (CRUD) operations that change the DID Documents are to be submitted to an IOTA Tangle in order to be publicly avaliable. They will either have to be a valid Int DID message or Diff DID message, submitted on the correct indexation for the identity. 
+
+### Create
+
+To generate a new DID, the method described in [generation](#generation) must be followed. A basic DID Document must be created that includes the public key used in the DID creation process as an `verificationMethod`. This DID Document must be formatted as an Integration DID message and published to an IOTA Tangle on the indexation generated out of the public key used in the DID creation process. 
+
+### Read
+
+To read the latest DID Document associated with a DID, the following process must be executed:
+* Query all the Integration DID messages from the indexation, which is the `tag` part of the DID.
+* Order the messages based on `previous_message_id` linkages. See [Determining Order](#determining-order) for more details.
+* Validate the first Integration DID message as containing a public key inside the `verificationMethod` field that, when hashed using the `BLAKE2b-256` hashing algorithm, equals the `tag` field of the DID. 
+* Verify the signatures of all the DID messages. Signatures must be signed using a public key avaliable in the previous DID message. 
+* Ignore any messages that are not signed correctly, afterwards ignore any messages that are not first in the ordering for their specific location in the chain.
+* If a URL parameter is added to the Resolution of `diff=false`, the following steps can be skipped and you should now have the latest DID Document.
+* Query all the Differentiation DID messages from the indexation, generated by the MessageId from the last valid Integration DID message, hashed using `Blake2b-256` and encoded in `hex`.
+* Order and validate signatures as similarly as to the Integration DID messages.
+* Ignore messages with illegal operations such as removing or updating a signing key.
+* Apply all valid Differentation updates to the state generated from the Integration DID messages. This will provide you with the latest DID Document.
+
+#### Determining Order
+
+To determine order of any DID messages, the following algorithm must be applied:
+* Order is initially established by recreating the chain based on the `previous_message_id` linkages. 
+* When two or more Messages compete, an order must be established between them.
+* To determine the order, check which milestone confirmed the messages.
+* If multiple messages are confirmed by the same milestone, we order based on the IOTA MessageId with alphabetical ordering.
+
```

Comment:
Can I attack this by farming a diff did message that starts with an "a"? How will this work in coordicide?

Reply:
> Can I attack this by farming a diff did message that starts with an "a"? 

No, since only the DID holder has the private key to sign a valid DID message. Your attack will be ignored during the resolving process. 

> How will this work in coordicide?

¯\_(ツ)_/¯ (We will cross that bridge when we get there)